php/Blocks/Loader.php

```diff
@@ -330,9 +330,7 @@ protected function render_block_template( $block, $attributes, $content ) {
 		$type = 'block';
 
 		// This is hacky, but the editor doesn't send the original request along.
-		$context = filter_input( INPUT_GET, 'context' );
-
-		if ( 'edit' === $context ) {
+		if ( 'edit' === filter_input( INPUT_GET, 'context' ) ) {
```

Comment:
Similar to above, because `filter_input()` doesn't sanitize, it shouldn't be stored in the variable `$context`.

`filter_input()` should simply run inline.